Source/RealtimeMeshComponent/Public/RealtimeMeshSimple.h

```diff
@@ -348,7 +348,7 @@ namespace RealtimeMesh
 
 		virtual void ProcessEndOfFrameUpdates() override;
 
-		friend class URealtimeMeshSimple;
+		friend class ::URealtimeMeshSimple;
```

Comment:
Explicit namespace is necessary to prevent this from being treated as a different type

Reply:
Also fixed in https://github.com/TriAxis-Games/RealtimeMeshComponent/pull/292/files, but I think it's better to fix the friend declaration